src/window/window_wrapper.rs

```diff
@@ -158,18 +161,44 @@ impl WinitWindowWrapper {
         self.windowed_context.window().set_ime_allowed(ime_enabled);
     }
 
+    pub fn set_os_blur(&mut self, os_blur: Option<Color>) {
+        let window = self.windowed_context.window();
+        self.os_blur = os_blur;
+        let result = if let Some(blur_color) = self.os_blur {
+            let blur_color = (
+                blur_color.r(),
+                blur_color.g(),
+                blur_color.b(),
+                blur_color.a(),
+            );
+            #[cfg(target_os = "windows")]
+            window_vibrancy::apply_acrylic(window, Some(blur_color))
+        } else {
+            #[cfg(target_os = "windows")]
+            window_vibrancy::clear_acrylic(window)
```

Comment:
[It looks like macOS could be supported as well](https://docs.rs/window-vibrancy/0.4.2/window_vibrancy/index.html#example), although we'd need with a macOS machine to confirm that.

Reply:
Yeah I mentioned this in the issue. I dont know how the macos arguments work and it was late so I just didnt bother. This PR isn't really mergeable as is and should probably be done with a winit upstream change anyway. I just threw it together as an example